homework/list.js

```diff
@@ -1,79 +1,81 @@
 'use strict';
 
-(function() {
+(function(exports) {
+  var list = function() {
+    var _listNoteContent = [];
+    var _wrapper = null;
+  };
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+  list.prototype = {
+    start() {
+      this._wrapper = document.querySelector('#note-list-wrapper');
+      this.fetchList().then((function(data) {
```

Comment:
Good, although I would recommend it to use Arrow function instead of `bind` API.